.ci/scripts/send-pr.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+set -uexo pipefail
+
+readonly APM_AGENT=${1}
+readonly APM_AGENT_SPECS_DIR=${2}
+readonly APM_AGENT_REPO_NAME="apm-agent-${APM_AGENT}"
+
```

Comment:
```suggestion
readonly GIT_DIR=.ci/git

```

Reply:
BTW, there is a missing closing } in the variable. Fixing it in 76989531a840ed096ff306a6caf78e31c1b82030